dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -1,9 +1,12 @@
 - content_for(:head) do
   - if @level.is_a?(Blockly) && !@level.uses_droplet?
-    %script{src: webpack_asset_path('js/blockly.js')}
-    %script{src: webpack_asset_path('js/en_us/blockly_locale.js')}
-    %script{src: webpack_asset_path('js/en_us/common_locale.js')}
-    %script{src: webpack_asset_path("js/en_us/#{@level.game.app}_locale.js")}
+    - if @level.is_a?(Dancelab) || @level.is_a?(Poetry)
+      %script{src: webpack_asset_path('js/googleblockly.js')}
+    - else
+      %script{src: webpack_asset_path('js/blockly.js')}
+      %script{src: webpack_asset_path('js/en_us/blockly_locale.js')}
+      %script{src: webpack_asset_path('js/en_us/common_locale.js')}
+      %script{src: webpack_asset_path("js/en_us/#{@level.game.app}_locale.js")}
```

Comment:
Thanks for this question. I was following suit of @molly-moen 's [PR](https://github.com/code-dot-org/code-dot-org/pull/49798/files). Tagged you on the question [here](https://github.com/code-dot-org/code-dot-org/pull/49798/files#r1203135508).

Reply:
I'll go ahead and move it back for now. I wonder if the files are needed at all since this is for levelbuilder and we don't need localization?

Reply:
FYI - I removed and these files are necessary for CDO blockly - rendering of blocks did not occur correctly without them on a CDO Blockly level when editing. They don't seem necessary for Google blockly on the levels I edited but leaving where they are since we haven't pinpointed why.